File: Source/WickedSick.KineticGraph.Test/TestEdge.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using WickedSick.KineticGraph.Controls;

namespace WickedSick.KineticGraph.Test
{
    public class TestEdge : IEdge
    {
        public TestEdge(ILinkable source, ILinkable sink)
        {
            Source = source;
            Sink = sink;
        }

        public ILinkable Source { get; protected set; }
        public ILinkable Sink { get; protected set; }
    }
}

File: Fayde/testsite/Models/TestEdge.ts
import ILinkable = Fayde.KineticGraph.ILinkable;

class TestEdge {
    Source: ILinkable;
    Sink: ILinkable;
    constructor(source: ILinkable, sink: ILinkable) {
        this.Source = source;
        this.Sink = sink;
    }
}
export = TestEdge;